spec/features/hub/cache_control_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe "Cache control" do
+  let(:user) { create :admin_user }
+
+  before do
+    login_as user
+  end
+
+  context "user on profile page, logs out and clicks back button", js: true do
+    it "cache is cleared and redirects to the login page" do
+      visit hub_user_profile_path
+      click_on 'Sign out'
+      page.evaluate_script('window.history.back()')
+      expect(page).to have_text "Sign in"
+    end
+  end
```

Comment:
Question for my own learning: Were you able to recreate the reported bug with this test before adding the no-cache headers?

Reply:
Yes! Good question, the tests failed when I commented out the set_cache_headers method